### PR TITLE
add mass and energy conservation check

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Partially addresses #1079. Right now it only works when radiation is the only external tendency, as all other external tendencies change within one timestep. @simonbyrne suggests that a better way is to add some state variables to accumulate fluxes. I still would like to merge this in as a temporary solution though, as I think it would be good to make sure we are conserving mass and energy in ci.

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]
Code changes look fine to me

File: examples/hybrid/callbacks.jl [hunk 1]
```diff
@@ -150,6 +160,23 @@ function turb_conv_affect_filter!(integrator)
     ODE.u_modified!(integrator, false)
 end
 
+horizontal_integral_at_boundary(f, lev) = sum(
+    Spaces.level(f, lev) ./ Fields.dz_field(axes(Spaces.level(f, lev))) .* 2,
```
Comment (charleskawczynski) [inline examples/hybrid/callbacks.jl:164]: Can you remind me why the `*2` is needed here?

Comment (szy21) [reply examples/hybrid/callbacks.jl:164]: Because at the boundary faces, `sum` calculates the integral over dz_field/2.

Comment (charleskawczynski) [reply examples/hybrid/callbacks.jl:164]: Ah, right, thanks!

**Pull request merged**